src/IdealizedSimulations/neverworld_simulation.jl

```diff
@@ -0,0 +1,409 @@
+# Neverworld
+#
+# Ingredients:
+#
+#   * Zonally-periodic domain with continental shelves on all boundaries except Southern Ocean
+#       * longitude = (0, 60)
+#       * 2 configurations in latitude
+#           - Half basin: latitude = (-70, 0)
+#           - Full basin: latitude = (-70, 70)
+#       * z to -4000m
+#       * Southern Ocean channel from -60 to -40 (with a ridge to -2000 m)
+#
+#   * Zonally-homogeneous wind stress with mid-latitude jet and trade winds
+#
+#   * Buoyancy 
+#       * restoring hot at the equator and cold at the poles (parabola, cosine, smooth step function)
+#       * equator-pole buoyancy differential: 0.06 (α * g * 30 ≈ 0.06 with α=2e-4, g=9.81)
+#       * exponential initial vertical stratification with N² = 6e-5 and decay scale h = 1000 m
+#           - eg bᵢ = N² * h * exp(z / h)
+#
+#   * Quadratic bottom drag with drag_coefficient = 1e-3
```

Comment:
should these be part of the docstring of `neverworld_simulation()`?

Reply:
We can though I'm not a huge fan of massive docstrings, I think they are not user-friendly...

Better to put them in legit documentation which we don't have yet.

Reply:
OK.

Docstrings could be either user friendly or not, depending on how they are written.

As a user of any repo I feel it's really user-friendly if I do `help?> a_method` and I get back a useful, meaningful, docstring with some examples that make sense.

Reply:
Of course I'm in favor of doctstrings, but I don't think we should put anything longer than a page in a docstring...

Reply:
I think the baseline information most people need is: what are the function arguments. If concise examples can be provided then that is excellent. I'm less sure about docstrings that span multiple pages. For example we wouldn't load an entire oceananigans example into a docstring for `Simulation`. There's a limit beyond which the excess information is hurting not helping